packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -104,7 +104,7 @@ export const RadioButton = compose<IRadioButtonType>({
   settings,
   slots: {
     root: View,
-    button: { slotType: View, filter: filterViewProps },
+    button: { slotType: View },
```

Comment:
Does filterViewProps break stuff on macOS? I think that's one of those things we never stood up on macOS that we probably should. I'm sure removing that breaks things for other platforms...

Reply:
No, filterViewProps doesn't break stuff on macOS. I had to remove it from button to allow the focus ring to land on the radio button. I tested my change on win32 and it was working fine, not sure about the other platforms though...

Reply:
I think only teasing win32 is fine for this control, since it requires FocusZone which is only on these two platforms. 
Still, maybe we should reach out to someone in CXE to make sure it's OK to remove this?

Reply:
Is removing filterViewProps only necessary to add a feature on macOS? If so, it doesn't seem like a good pattern to remove stuff from one platform to add something to a different platform. 

Filtering out view props is a common pattern in all our controls for slots that are of type View (excluding the root). It feels strange to remove this here. Perhaps dividing these into .platform files would be better?

RadioButton.win32.tsx and RadioButton.macOS.tsx?

@ksiler @PPatBoyd @lenahong What are your thoughts on this?

Reply:
@chiuam how did `filterViewProps` prevented the focus ring from landing on the radio button? I'd like to understand that more. 

I agree that we shouldn't be removing stuff just to add something to one platform. Even though it doesn't break anything now, we might want to use platform specific props in the future. 

@Saadnajmi / @chiuam  Is it possible to implement a filterViewProps for macOS? How hard would that be? Seems like react-native-macos has View, Text, and Image components here: https://github.com/microsoft/react-native-macos/tree/master/Libraries Is assume this is where you'd pull the platform specific props from?

Reply:
Would that just be making an `adapters.macos.ts` file here? https://github.com/microsoft/fluentui-react-native/tree/907fea51098ba59a8235128e17fb82a327b7adb2/packages/utils/adapters/src

It might help if we had a simple overview of what filterviewprops/adapters was doing or why it's necessary. I assume it's because the props passed to the core components (View, Text, Image) can be different on each platform (especially out of tree platforms), and so this... does something to help that? Does it exclude props? How do we stop it from excluding a prop that may be necessary, like `enableFocusRing`? That's one of the things I've been confused about :)

Reply:
Yeah I believe so.

Your assumption is my understanding as well. The adapter file allows you to pull in platform specific props and define them through the masks (for View, Text, and Image) and it can exclude props if you desire. I know for win32, we exclude some android and ios specific props and include win32 specific props. Does that answer your question?

It sounds like `enableFocusRing` is a macOS specific prop? Where does it come from/where is it defined?